Tests/BSONTests/BSONCorpusTests.swift

```diff
@@ -3,6 +3,7 @@ import Foundation
 import Nimble
 import XCTest
 
+@available(OSX 10.13, *)
```

Comment:
why did we need to add this?

Reply:
It was there because I originally wrote my regexes using named capture groups which need that annotation, I went to just using the index since it got frustrating that the whole call chain such an annotation, and that's something we probably wouldn't want to pass along to users. 
Anyway we don't need it anymore, good catch!